core/exception/interrupt_spec.rb

```diff
@@ -0,0 +1,35 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+
+describe "Interrupt.new" do
```

Comment:
Sounds good.
But, It seems duplicated with [here](https://github.com/ruby/spec/blob/5309c48d94203df3fd2d8808b8d0517d6dbfcfae/core/exception/exception_spec.rb#L27).
How do you think?